generic_tracers/generic_COBALT.F90

```diff
@@ -2332,6 +2334,7 @@ subroutine generic_COBALT_update_from_source(tracer_list,Temp,Salt,rho_dzt,dzt,h
     real :: rev_angle, dec_angle, temp_arg
 
     logical ::  phos_nh3_override
+    logical ::  pha_all_same 
```

Comment:
Should we set this default to false?

Reply:
sure, I added default false - do you think true is better?

Reply:
Hmm as a second thought, true might be a better option - assume it is always a fixed variable array unless we turn on calculation.